plugins/nodes/src/js/columns/NodesTableCPUBarColumn.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import Node from "#SRC/js/structs/Node";
+
+// TODO: DCOS-39079
+// import { IWidthArgs as WidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
+import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
+
+export function cpubarRenderer(data: Node): React.ReactNode {
+  // TODO: DCOS-38822
+  return <span>{data.get("used_resources").cpus.toString()}</span>;
+}
+export function cpubarSizer(args: WidthArgs): number {
+  // TODO: DCOS-38822
+  return Math.max(100, args.width / args.totalColumns);
```

Comment:
I know there is the TODO, but I could see lots of cases where the column width isn't super important and would be fine with a default.  I wonder if maybe something for the table component to support a smart default, much like you have implemented here and in all the following "classes"?

Reply:
TODO isnt related to a default value. we could think of something like that, but It would be a feature/change in ui kit. Also I think @weblancaster and @Poltergeist should be involved in that discussion. <Adding a JIRA, will edit it here>

Reply:
What is a smart default on a infinite width? I mean we could go with `totalwidth/numberofcolumns` Personally I think this is very opinionated. But I am okay with adding this to the table.